src/plugins/intel_gpu/tests/unit/fusions/resample_fusion_test.cpp

```diff
@@ -348,3 +348,27 @@ INSTANTIATE_TEST_SUITE_P(fusings_gpu, resample_eltwise_fusing_through_not_allowe
 
     resample_test_params{ CASE_RESAMPLE_FP16_1, RESAMPLE_ELTWISE_FUSING_THROUGH_CNT },
 }));
+
```

Comment:
nit: Could you add test case for target pattern?
Target pattern should be dynamic shape, block format and resample fusion for verifying resample shape dependency.